### DISCUSSION TITLE
GitHub copies wrong code to clipboard

### DISCUSSION THREAD

**Discussion by xiaohk** [community/community#45272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I copy code from the new code viewer, if a line contains a highlighted text, then this line would be duplicated in the clipboard. See the attached video for detail.

https://user-images.githubusercontent.com/15007159/214415860-a9f814a0-e5bc-4aaf-a3dd-1e29fa81a578.mp4

**Comment by colinwm** [comment]
Sorry about that! Will fix it

**Reply by pcjmfranken** [reply]
FYI Copying selected text with keyboard cmd+c no longer works in the new code view

**Reply by colinwm** [reply]
Sorry about that, it's now fixed

**Reply by pcjmfranken** [reply]
Woah that was quick, cheers! 🚀
